### DISCUSSION TITLE
(vlc) omit in title "install" for virtual package?

### DISCUSSION THREAD

**Discussion by J-Arx** [chocolatey-community/chocolatey-packages#1896] [Ideas]
In this [vlc](https://github.com/chocolatey-community/chocolatey-packages/tree/master/automatic/vlc) [virtual package](https://github.com/chocolatey-community/chocolatey-packages/commit/2013aac73cb4802b83c3ab06ca16c5e402ccb36f), is it due to file "vlc.nuspec" line 6 `<title>VLC media player (Install)</title>` that "**(Install)**" which causes the [package search](https://community.chocolatey.org/packages) "vlc media player" lists two times "VLC media player **(Install)** 3.0.17.4"?
If so, can somebody help to omit "**(Install)**" in that virtual package title?
Thanks.

**Comment by AdmiringWorm** [comment]
This can be considered a bug, and could have been opened directly as a issue instead of a discussion.

I'll create an issue based on this discussion.

If you are interested in submitting a PR to fix the title it would be appreciated, it is just a matter of changing the title in the nuspec file: https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/vlc/vlc.nuspec#L6

Do note it will only be changed for versions submitted after the change is pulled into the repository.
